package.json

```diff
@@ -40,7 +39,8 @@
     "husky": "^8.0.3",
     "lint-staged": "^15.2.0",
     "prettier": "^3.1.1",
-    "vite": ">=5.0.12",
+    "sass": "^1.70.0",
```

Comment:
Why did you install sass in "devDependencies"  instead "dependencies". Look at the latest changes in dev branch and get it.